### ISSUE TITLE
Species compendium-specific README

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#983]
### Context

The README that will be delivered along with a species compendium is currently the same as any ol' smasher job.

### Problem or idea

The README should be tailored to this. I will also construct my QC checks of the zebrafish compendium that will come from #888 with the possible inclusion of a Jupyter notebook that does these QC checks (e.g., would be useful to include things like PCA). 

### Solution or next step

Related to: AlexsLemonade/refinebio-docs#22

Next step is performing QC checks

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by jaclyn-taroni** [comment]
Realistically, this is blocked by having a "test" species compendium that is more representative of what will happen in production (vs. the very small, atypical compendium I have been working with) and that has metadata: https://github.com/AlexsLemonade/refinebio/pull/888#issuecomment-457386447

**Comment by jaclyn-taroni** [comment]
I have a more representative zebrafish compendium and QC is currently in progress.

**Comment by jaclyn-taroni** [comment]
Closed by #1238
